### PR TITLE
DOCS-319 - Alerts/Monitors tags

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

alerts/monitors tags

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

https://sumologic.atlassian.net/browse/DOCS-319

### CODE REVIEW

File: docs/api/organizations-management.md [hunk 1]
```diff
@@ -41,4 +41,5 @@ Check out our additional [Organizations Management API endpoints](https://organi
 
 * Organizations (all role capabilities)
 
--->
\ No newline at end of file
+-->
+
```
Comment (kimsauce) [inline docs/api/organizations-management.md:45]:
```suggestion
--> 
```

File: docs/alerts/monitors/settings.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ The **Monitors** page allows you to view, create, manage, and organize your moni
 [**Classic UI**](/docs/get-started/sumo-logic-ui-classic). To access the **Monitors** page, in the main Sumo Logic menu select **Manage Data > Monitoring > Monitors**.  
 
 [**New UI**](/docs/get-started/sumo-logic-ui/). To access the Monitors page, in the main Sumo Logic menu select **Alerts > Monitors**. You can also click the **Go To...** menu at the top of the screen and select **Monitors**.
- 
+
```
Comment (kimsauce) [inline docs/alerts/monitors/settings.md:15]:
```suggestion
 
```

File: docs/api/organizations-management.md [hunk 3]
```diff
@@ -41,4 +41,4 @@ Check out our additional [Organizations Management API endpoints](https://organi
 
 * Organizations (all role capabilities)
 
--->
\ No newline at end of file
+-->
```
Comment (kimsauce) [inline docs/api/organizations-management.md:44]:
```suggestion
-->
```

File: docs/manage/users-roles/roles/role-capabilities.md [hunk 4]
```diff
@@ -101,9 +101,9 @@ Folder-level permissions are available if your org has fine-grained Monitor perm
...
+| View Monitors	| If [Monitors folder permissions](/docs/alerts/monitors/settings/#monitor-folder-permissions) are enabled for your org, users with this capability can view folders on the [Monitors](/docs/alerts/monitors) page to which they've been granted View access, and the Monitors contained in those folders.|
+| Manage Monitors	| Users with this capability can create new folders and [Monitors](/docs/alerts/monitors), and grant other roles permissions to the folders they create. If [Monitors folder permissions](/docs/alerts/monitors/settings/#monitor-folder-permissions) are enabled for your org, users with this capability can also create, edit, delete, update and grant permissions to folders to which another user has granted them those permissions.|
+| Admin Monitors	| If [Monitors folder permissions](/docs/alerts/monitors/settings/#monitor-folder-permissions) are enabled for your org, users with this capability have full access (Create, Edit, Delete, Update, and grant permissions) to ALL folders and monitors on the Monitors page. This is similar to the Content Administrator capability of the Content Library.|
```
Comment (jpipkin1) [inline docs/manage/users-roles/roles/role-capabilities.md:106]:
```suggestion
| View Monitors	| If [monitors folder permissions](/docs/alerts/monitors/settings/#monitor-folder-permissions) are enabled for your org, users with this capability can view folders on the [Monitors](/docs/alerts/monitors) page to which they've been granted View access, and the Monitors contained in those folders.|
| Manage Monitors	| Users with this capability can create new folders and [monitors](/docs/alerts/monitors), and grant other roles permissions to the folders they create. If [monitors folder permissions](/docs/alerts/monitors/settings/#monitor-folder-permissions) are enabled for your org, users with this capability can also create, edit, delete, update and grant permissions to folders to which another user has granted them those permissions.|
| Admin Monitors	| If [monitors folder permissions](/docs/alerts/monitors/settings/#monitor-folder-permissions) are enabled for your org, users with this capability have full access (Create, Edit, Delete, Update, and grant permissions) to ALL folders and monitors on the Monitors page. This is similar to the Content Administrator capability of the Content Library.|
```

**Review by jpipkin1 — APPROVED** [review]
Looks good, @kimsauce.

**Pull request merged**